homeassistant/components/sensor/wunderground.py

```diff
@@ -29,50 +29,562 @@
 
 DEFAULT_LANG = 'EN'
 
-MIN_TIME_BETWEEN_UPDATES_ALERTS = timedelta(minutes=15)
-MIN_TIME_BETWEEN_UPDATES_OBSERVATION = timedelta(minutes=5)
+MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=5)
```

Comment:
yes, in fact with this change all the different API requests (e.g. alerts, conditions, forecasts,... etc.) are combined into single one according to these instructions: https://www.wunderground.com/weather/api/d/docs?d=data/index#examples
So in fact this component will not do more than 288 API calls per day (for single instance)